modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -559,9 +559,16 @@ validateCalculatedResults <- function(calculatedResults, dryRun, curveNames, tes
       }
     } else {
       for (row in 1:nrow(newBatchIds)) {
+        NOT_REGISTERED_MESSAGE <- paste0(mainCode, " '", newBatchIds$Requested.Name[row],
+                          "' has not been registered in the system.")
+        
+        # TODO: Put this behind the unique alias config when we add it to ACAS
+        if(applicationSettings$server.service.external.preferred.batchid.allowParentAliasLotNames && mainCode == "Corporate Batch ID") {
+          NOT_REGISTERED_MESSAGE <- paste0(NOT_REGISTERED_MESSAGE, " If the ", mainCode, " is a Parent Alias Lot Name it may also mean that the Parent Alias is referenced by zero or multiple Parents.")
```

Comment:
I find this error message unnecessarily confusing. Most of the time I'd expect us to get an error purely because the parent alias isn't registered. The "zero or multiple" wording is especially confusing to me. I'm inclined to leave this expanded error message out, even though it would be helpful in that one scenario where a parent alias is duplicated across different parents.

Reply:
Thanks, I'm just going to update this message per your feedback but leave it in.  The reason being that I have a TODO as part of the global parent alias uniqueness config to hide this message when that config is set to true as ACAS shouldn't allow multiple aliases.